.goreleaser.yml

```diff
@@ -1,287 +1,55 @@
+# https://goreleaser.com/customization/changelog
+changelog:
+  # Disable posting of the changelog.
+  skip: true
+
+checksum:
+  # Tag will always be v.0.0.0 so we provide a name template to achieve the
+  # desired format.
+  name_template: boulder-release-{{ time "2006-01-02"  }}-checksum.txt
+
+# https://goreleaser.com/customization/archive
+archives:
+  # Only the binary will be uploaded.
+  - format: binary
+    # Tag will always be v.0.0.0 so we provide a name template to achieve the
+    # desired format.
+    name_template: boulder-release-{{ time "2006-01-02"  }}_{{ .Os }}_{{ .Arch }}
+    replacements:
+      amd64: x86_64
+
+# https://goreleaser.com/customization/nfpm
+nfpms:
+  - id: boulder-nfpm
+    package_name: boulder
+    # Tag will always be v.0.0.0 so we provide a name template to achieve the
+    # desired format.
+    file_name_template: boulder-release-{{ time "2006-01-02" }}_{{ .Os }}_{{ .Arch }}
+    builds:
+      - boulder-build
+    maintainer: boulder <boulder@letsencrypt.org>
```

Comment:
Is this a real email address? I don't have any problem listing it here, just curious if it'll actually show up anywhere.